src/ExceptionTransformer.ts

```diff
@@ -46,9 +48,80 @@ class ExceptionTransformer {
         exception.fallback_message
       );
     }
-
     return exceptionMap;
   }
+
+  public generateSpesificFieldError(errorInfo: Exception | null | undefined) {
+    const detail =
+      errorInfo && typeof errorInfo.detail === "object" && errorInfo.detail;
+
+    if (detail) {
+      return function getError(fieldName: string) {
+        return detail[fieldName];
+      };
+    } else {
```

Comment:
Else block here is redundant
```suggestion
       ```
       if (detail) {
         return ...
       }

       return () => undefined;
       ```
```